src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -27,15 +27,15 @@ redirects:
 
 import alertsAlertsandAiOverviewPage from 'images/alerts_screenshot-full_alerts-and-ai-overview-page.png'
 
-As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening where, and what to do about it. New Relics alerts and applied intelligence were designed to end response fatigue so your team can not only detect and diagnose incidents, but prioritize them for immediate action.
+As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening where, and what to do about it. New Relic's alerts and applied intelligence were designed to end response fatigue so your team can not only detect and diagnose incidents, but prioritize them for immediate action.
```

Comment:
I like this a lot. One very small thing, the "what's happening where, and what to do about it" part in this sentence could be expanded to "what's happening, where it's happening, and what to do about it." This is super small though and just a little consideration.